app/serializers/rest/account_serializer.rb

```diff
@@ -20,6 +20,26 @@ def value
 
   has_many :fields
 
+  class AccountIdentityProofSerializer < ActiveModel::Serializer
+    attributes :provider, :provider_username, :updated_at, :proof_url, :profile_url
+
+    def proof_url
+      object.badge.proof_url
+    end
+
+    def profile_url
+      object.badge.profile_url
+    end
+
+    def provider
+      object.provider.capitalize
+    end
+  end
+
+  has_many :identity_proofs do
```

Comment:
We include a full account entity along with every status entity, so I believe this is not an option, performance-wise. It's an n+1 query and won't be used in most requests. I think it should be fetchable through a separate API endpoint.

Reply:
yup. makes sense. i took a stab at it. lemme know what you think.